platform/kube/inject/inject.go

```diff
@@ -40,7 +40,9 @@ import (
 // Defaults values for injecting istio proxy into kubernetes
 // resources.
 const (
+	DefaultHubEnvVar       = "MANAGER_HUB"
```

Comment:
`platform/kube/inject/inject.go` doesn't use `Default{Tag,Hub}EnvVar` at all so I'm not sure it makes sense to define them in this file. Probably better to keep that in `cmd/istioctl/inject.go`? In hindsight the DefaultHub and DefaultTag should also probably be under cmd/ instead of platform/